### PR TITLE
build: Install Inertia for Front-End development

### PR DESCRIPTION
**Pull request by AxonC**

### CODE REVIEW

File: public/mix-manifest.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (atoff) [inline public/mix-manifest.json]: We should probably .gitignore this?

File: webpack.mix.js [hunk 2]
```diff
@@ -11,5 +11,9 @@ const mix = require('laravel-mix');
  |
  */
 
-mix.js('resources/js/app.js', 'public/js')
-    .sass('resources/sass/app.scss', 'public/css');
+
+mix.js('resources/js/app.js', 'public/js').webpackConfig({
+    resolve: {
+        alias: {'vue$': 'vue/dist/vue.runtime.js'}
```
Comment (atoff) [inline webpack.mix.js]: What is this for?

Comment (AxonC) [reply webpack.mix.js]: allows us to use @vue

Comment (AxonC) [reply webpack.mix.js]: Removed

File: public/mix-manifest.json (hunk 1 above)
Comment (AxonC) [reply public/mix-manifest.json]: removed & ignored

**Review by atoff — DISMISSED** [review]

**Review by atoff — APPROVED** [review]

**Pull request merged**